apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3158,7 +3160,77 @@ static void tc_libc_stdio_zeroinstream(void)
 
 	TC_SUCCESS_RESULT();
 }
+/**
+* @testcase         tc_fs_driver_mtd_procfs_ops
+* @brief            mtd procfs ops
+* @scenario         opens /proc/mtd and performs operations
+* @apicovered       mtdprocfs_operations (mtd_open, mtd_dup, mtd_stat, mtd_close)
+* @precondition     NA
+* @postcondition    NA
+*/
+#if !defined(CONFIG_FS_PROCFS_EXCLUDE_MTD)
+static void tc_driver_mtd_procfs_ops(void)
+{
+	int fd1;
+	int fd2;
+	int ret;
+	struct stat st;
+
+	fd1 = open(MTD_PROCFS_PATH, O_RDONLY);
+	TC_ASSERT_GEQ("open", fd1, 0);
+
+	fd2 = dup(fd1);
+	TC_ASSERT_GEQ_CLEANUP("dup", fd2, 0, close(fd1));
+
+	ret = stat(MTD_PROCFS_PATH, &st);
```

Comment:
@kavyasrinunna 
 Please check if "st" also needs to be verified

Reply:
I think "st" may not be  null, so verifying "st" may not be required.